cloud-init/write-archive-artifact-data-to-bucket.tpl.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+# This is a Terraform template file, and the input variables are
+# passed in via templatefile().
+#
+# shellcheck disable=SC2154
+
+# This script is used write out a bash script that can be used to create an
+# assessment artifact archive and copy it to the appropriate S3 bucket.
+
+# Input variables are:
+# * artifact_export_bucket_name - the name of the assessment artifact export S3
+#   bucket
+# * artifact_export_path - the path to copy the artifact to in the S3 bucket
+# * assessment_id - the identifier for the assessment
+# * permissions - the octal permissions to assign the script
```

Comment:
Is there a reason why these permissions must be specified in octal?  Looking at the script it appears that any format supported by `chmod` should be OK.

Reply:
I'm following the same pattern that you started in #207:
https://github.com/cisagov/cool-assessment-terraform/blob/840cbc636c801e9812630eb08c75c89799d00b4d/kali_cloud_init.tf#L66-L76

If we change this, we should change it everywhere in this repo.

Reply:
Unless you see a reason why it has to be octal I'd prefer that you change yours.  I can create an issue for the rest.

- Geez, if I walked to the middle of the Golden Gate and jumped off would you still follow me?
- Do as I say, not as I do.

Reply:
I'm only suggesting changing the description to something like `the permissions to assign the script, specified in either the octal or symbolic formats understood by chmod`.  There is no need to change the actual values passed in.

Reply:
Please take a look at https://github.com/cisagov/cool-assessment-terraform/pull/213/commits/674c6efd7485bfdddddc4a82e9b1eba7f40e427c.

Reply:
Thank you!